<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_PixelX">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PixelY">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>-10</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="m_widget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>1021</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_draw">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Нарисовать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PixelX1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PixelY1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Пиксель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дуга</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прямоугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Круг</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эллипс</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Многоугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ломанная линия</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_stAngle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_spAngle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_an1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Нач.угол</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_an2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Угол охвата</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_pen">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Без карандаша</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сплошная линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пунктирная линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Точечная линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пунктирно-точечная линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Точечно-пунктирная линия</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Pen">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Стиль Карандаша</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_brush">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Без заливки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сплошная заливка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Плотная заливка 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Плотная заливка 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Плотная заливка 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Плотная заливка 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Плотная заливка 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Плотная заливка 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Плотная заливка 7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Горизонтальная заливка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вертикальная заливка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сетчатая заливка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Обратно-диагональная заливка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прямо-диагольная заливка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Диагонально-сетчатая заливка</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_brush">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Стиль заливки</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_penW">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_PenW">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Толщина Карандаша</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>25</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>41</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>140</y>
      <width>41</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>1021</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>1021</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_w">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>110</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет фона</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>620</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>m_widget</class>
   <extends>QWidget</extends>
   <header location="global">m_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit_PixelX</tabstop>
  <tabstop>lineEdit_PixelY</tabstop>
  <tabstop>lineEdit_PixelX1</tabstop>
  <tabstop>lineEdit_PixelY1</tabstop>
  <tabstop>lineEdit_stAngle</tabstop>
  <tabstop>lineEdit_spAngle</tabstop>
  <tabstop>comboBox_pen</tabstop>
  <tabstop>horizontalSlider_penW</tabstop>
  <tabstop>comboBox_brush</tabstop>
  <tabstop>pushButton_draw</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
